client/query_test.go

```diff
@@ -57,7 +57,7 @@ func (s *IntegrationTestSuite) TestQueryABCIHeight() {
 			res, err := clientCtx.QueryABCI(req)
 			s.Require().NoError(err)
 
-			s.Require().Equal(tc.expHeight, res.Height)
+			s.Require().LessOrEqual(tc.expHeight, res.Height)
```

Comment:
Did the gRPC path change affect here in any way?

Reply:
This is the frequent data race problem in CI.
This problem was happening too often in this PR. So I fixed it.

Reply:
This change is rollbacked. I'll register the another PR about this changes. Thank you.